test/e2e/scale_test.go

```diff
@@ -102,17 +102,24 @@ func testScaleToWithin(t *testing.T, logger *logging.BaseLogger, scale int, dura
 	}
 
 	go func() {
-		if err := deployGrp.Wait(); err != nil {
+		err := deployGrp.Wait()
+		logger.Info("Finished service creation.")
```

Comment:
Do we need both log statements? Since either of the log statements below will equate to this one.